src/xls.rs

```diff
@@ -455,10 +461,57 @@ impl<RS: Read + Seek> Xls<RS> {
     }
 }
 
+/// https://learn.microsoft.com/en-us/openspecs/office_file_formats/ms-xls/4d6a3d1e-d7c5-405f-bbae-d01e9cb79366
+struct Bof {
+    /// Binary Interchange File Format
+    biff: Biff,
+}
+
+/// https://www.loc.gov/preservation/digital/formats/fdd/fdd000510.shtml#notes
+#[derive(Clone, Copy)]
+enum Biff {
+    Biff2,
+    Biff3,
+    Biff4,
+    Biff5,
+    Biff8,
+    // Used by MS-XLSB Workbook(2.1.7.61) or Worksheet(2.1.7.61) which are not supported yet.
+    // Biff12,
+}
+
+/// BOF [MS-XLS] 2.4.21
+fn parse_bof(r: &mut Record<'_>) -> Result<Bof, XlsError> {
+    let mut dt = 0;
+    let biff_version = read_u16(&r.data[..2]);
+
+    if r.data.len() >= 4 {
+        dt = read_u16(&r.data[2..]);
+    };
+
+    let biff = match biff_version {
+        0x0200 | 0x0002 | 0x0007 => Biff::Biff2,
+        0x0300 => Biff::Biff3,
+        0x0400 => Biff::Biff4,
+        0x0500 => Biff::Biff5,
+        0x0600 => Biff::Biff8,
+        0 => {
+            if dt == 0x1000 {
```

Comment:
I wasn't able to find an example of `biff5` file that would trigger this logic. Please let me know if you'd like me to remove this code.

Reply:
Just to clarify: I'm talking about removing this code since, if we comment it out, as you've suggested earlier, we'd have no use for `dt` variable. Then I guess we'll need to comment out more code. I agree that following the spec is the best option, especially with no evidence that we'll ever encounter files that can trigger this logic.